script.js

```diff
@@ -1,77 +1,88 @@
-// Array to store the list of awesomeBooks
-let awesomeBooks = [];
-
-// Function to add a book
-function addBook() {
-  const bookTitle = document.getElementById('bookTitle').value;
-  const authorName = document.getElementById('authorName').value;
-  const errorMessage = document.querySelector('.feedback');
+class AwesomeBookManager {
+  constructor() {
+    this.awesomeBooks = [];
+    this.init();
+  }
 
-  if (bookTitle.trim() === '' || authorName.trim() === '') {
-    errorMessage.textContent = '❌ Add both "Title" & "Author"!';
-    return;
+  init() {
+    this.loadBooksFromLocalStorage();
+    document
+      .getElementById('add-btn')
+      .addEventListener('click', this.addBook.bind(this));
+    window.addEventListener('load', this.loadBooksFromLocalStorage.bind(this));
   }
 
-  const book = { title: bookTitle, author: authorName };
-  awesomeBooks.push(book);
+  // Function to add a book
+  addBook() {
+    const bookTitle = document.getElementById('bookTitle').value;
+    const authorName = document.getElementById('authorName').value;
+    const errorMessage = document.querySelector('.feedback');
 
-  document.getElementById('bookTitle').value = '';
-  document.getElementById('authorName').value = '';
-  /* eslint-disable */
-  updateBookList();
-  saveBooksToLocalStorage(); // Save the updated book list to local storage
-  /* eslint-disable */
-}
+    if (bookTitle.trim() === '' || authorName.trim() === '') {
+      errorMessage.textContent = '⚠️ Add both "Title" & "Author"!';
+      return;
+    }
 
-// Function to remove a book
-function removeBook(index) {
-  awesomeBooks = awesomeBooks.filter((_, i) => i !== index);
-  /* eslint-disable */
-  updateBookList();
-  saveBooksToLocalStorage(); // Save the updated book list to local storage
-  /* eslint-disable */
-}
+    const book = { title: bookTitle, author: authorName };
```

Comment:
- [ ] I will encourage you to add id because it is needed as required in the project requirement.
![image](https://github.com/efmomoh/awesome-books/assets/86200356/d1beb164-7d10-4138-a8df-f34e98d865f7)

Reply:
The author and title contain ID in the HTML and have been selected to style. Which ID are you referencing? Kindly explain @Olamarx